Website/src/rendering/src/components/SocialIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
+import { Field } from '@sitecore-jss/sitecore-jss-nextjs';
+
+type SocialIconProps = {
+  Link?: Field<string>;
+  Icon: IconProp;
+};
+
+export function SocialIcon(props: SocialIconProps): JSX.Element {
```

Comment:
Our other functional components are all exported with `default` to avoid using curly braces when importing them. Maybe we should continue using the same pattern for new components (this comment also applies to SpeakerInfoLine):

```
const SocialIcon = (props: SocialIconProps): JSX.Element => {
}

export default SocialIcon;
```